src/components/navBar/navBar.js

```diff
@@ -228,6 +228,11 @@ MdNavBarController.prototype._updateTabs = function(newValue, oldValue) {
         self._updateInkBarStyles(tab, newIndex, oldIndex);
       });
     }
+  } else {
+    var self = this;
+    this._$timeout(function(){
+      self._updateInkBarStyles(oldTab, -1, oldIndex);
+    });
```

Comment:
I think we can refactor to remove the need for a duplicated call to $timeout/updateInkbarStyler. How about (slightly more compact):

- get rid of the `if (newValue)` check (`tab` will be null)
- move the `$timeout` out of the `if(tab)` block
- change updateInkbarStyles to be able to handle a null tab